DependencyInjection/Configuration.php

```diff
@@ -163,7 +163,7 @@ private function configureClients(ArrayNodeDefinition $root)
                         // Make sure we only allow one of these to be true
                         return (bool) $config['flexible_client'] + (bool) $config['http_methods_client'] + (bool) $config['batch_client'] >= 2;
                     })
-                    ->thenInvalid('A http client can\'t be decorated with both FlexibleHttpClient and HttpMethodsClient. Only one of the following options can be true. ("flexible_client", "http_methods_client", "batch_client")')
+                    ->thenInvalid('A http client can\'t be decorated with several of FlexibleHttpClient, HttpMethodsClient and. Only one of the following options can be true. ("flexible_client", "http_methods_client", "batch_client")')
```

Comment:
yeah indeed. I worked on it, then got interrupted, and then I committed while forgetting that the message was incomplete. A line comment is much easier to understand that putting the info in the summary without any details 😄